tests/unittests/test_corrections.py

```diff
@@ -82,7 +82,7 @@ def test_calculate_k_med():
     assert test in expected
 
 def test_calculate_k_tab():
-    """Testa the table correction."""
+    """Test the table correction."""
     # Tests if correct k_tab value is returned from database
```

Comment:
Remove this comment and rename test to reflect this, e.g.

```python
def test_calculate_should_correctly_interpolate_table_correction_from_stored_values():
```

Reply:
This goes for all tests.

This is done so that you can see directly in the test report what is failing if a test fails

Reply:
done